tests/rt.conf

```diff
@@ -1,188 +1,188 @@
-
-#######################################################################################################################################################################################
-# CPLD tests
-#######################################################################################################################################################################################
-
-COMPILE | SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled,FV3_GFS_v16beta_coupled S2S=Y           | standard | - wcoss_cray gaea.intel jet.intel | fv3 |
-RUN     | cpld_control                                                                                                            | standard | - wcoss_cray gaea.intel jet.intel | fv3 |
-RUN     | cpld_restart                                                                                                            | standard | - wcoss_cray gaea.intel jet.intel |     | cpld_control
-RUN     | cpld_controlfrac                                                                                                        | standard | - wcoss_cray gaea.intel jet.intel | fv3 |
-RUN     | cpld_restartfrac                                                                                                        | standard | - wcoss_cray gaea.intel jet.intel |     | cpld_controlfrac
-
-RUN     | cpld_2threads                                                                                                           | standard | - wcoss_cray gaea.intel jet.intel |     |
-RUN     | cpld_decomp                                                                                                             | standard | - wcoss_cray gaea.intel jet.intel |     |
-RUN     | cpld_satmedmf                                                                                                           | standard | - wcoss_cray gaea.intel jet.intel | fv3 |
-RUN     | cpld_ca                                                                                                                 | standard | - wcoss_cray gaea.intel jet.intel | fv3 |
+COMPILE | DEBUG=Y                                                                                                                 | + wcoss_dell_p3                         | fv3 |
+COMPILE | DEBUG=Y                                                                                                                 | + wcoss_cray                            | fv3 |
+COMPILE | DEBUG=Y                                                                                                                 | + hera.intel                            | fv3 |
+COMPILE | SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta,FV3_GFS_v15p2_RRTMGP,FV3_GFS_v16beta_RRTMGP DEBUG=Y                                | + orion.intel                           | fv3 |
```

Comment:
I think I tested the compilation on Orion using just "DEBUG=Y' and it worked.

Reply:
Ok, I will make this change.